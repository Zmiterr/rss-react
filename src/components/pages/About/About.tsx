import React, { ReactElement } from 'react';

const About = (): ReactElement => {
	return (
		<div>
			<h1>Now you placed on About page</h1>
		</div>
	);
};

export default About;
