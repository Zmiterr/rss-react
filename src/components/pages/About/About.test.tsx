import React from "react";
import About from './About';
//test deps
import Enzyme, { shallow, render, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { BrowserRouter } from 'react-router-dom';


Enzyme.configure({ adapter: new Adapter() })


describe("About component", () => {
	it("should render About component", () => {
		const component = shallow(<BrowserRouter><About /></BrowserRouter>);
		expect(component).toMatchSnapshot();
	});
});
