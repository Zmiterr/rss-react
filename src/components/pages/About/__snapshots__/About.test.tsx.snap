// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render About component 1`] = `ShallowWrapper {}`;
